### DISCUSSION TITLE
AE中多个素材间连接无问题，播放正常，导出 pag后，插入任何素材，某些片段间会出现黑帧

### DISCUSSION THREAD

**Discussion by EngeWang** [Tencent/libpag#2561] [Q&A]
版本信息】
PAG SDK 版本 | 操作系统版本-mac 14.1 (23B2073) | PAGViewer 版本-4.3.6.102（stable) | AE版本-2024-24.1.0

多段视频素材拼接导出一个 pag文件，在PAGViewer中导入素材预览时部分片段黑帧

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在是否方便提供下问题对应的aep、pag文件，用于复现该问题

**Comment by jinwuwu001** [comment]
请问该问题是否还需要跟进，若需要请提供上述信息

**Comment by jinwuwu001** [comment]
用户长期无进一步反馈，暂时关闭，有需要可以再开启
